### PR TITLE
Add blog post on "How FerretDB fetches data (About query pushdown)"

### PR DESCRIPTION
**Pull request by Fashander**
# Description

Closes FerretDB/engineering#17

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [ ] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by noisersup — APPROVED** [review]
Thanks for translating it to markdown! I've left small question, and move it from draft. Feel free to draft it again if there's reason behind it

File: website/blog/2023-01-25-ferretdb-fetches-data-query-pushdown.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+---
+title: "How FerretDB fetches data - about query pushdown"
+slug: ferretdb-fetches-data-query-pushdown
+author: Patryk Kwiatek
+description: "Find out how FerretDB uses query pushdown to fetch data from the storage layer (also called as “backend”)"
+image: /img/blog/ferretdb-query-pushdown.jpg
+unlisted: true
```
[the remark this answers is not in the document]
Comment (Fashander) [reply website/blog/2023-01-25-ferretdb-fetches-data-query-pushdown.md:7]: It'll be used for all blog posts that hasn't passed through review. Mainly, it means the new post can be merged to repo without it showing up in production. That gives us enough room to decide the time to publish it and make changes that way.

[the remark this answers is not in the document]
Comment (noisersup) [reply website/blog/2023-01-25-ferretdb-fetches-data-query-pushdown.md:7]: Makes sense, although I thought that we decide whether the blog is updated or not with approving dependabot merge

[the remark this answers is not in the document]
Comment (Fashander) [reply website/blog/2023-01-25-ferretdb-fetches-data-query-pushdown.md:7]: That's still in place. But sometimes we can have more than 2 completed and approved posts that's not scheduled for publishing until later. We want to have them merged into the main pending when we decide to have it appear in production

**Review by rumyantseva — APPROVED** [review]
:shipit:

**Review by chilagrow — APPROVED** [review]
Let's publish this!

**Review by w84thesun — COMMENTED** [review]
Nice! Added small fixes to make the `active` parameter bool everywhere through the text.

File: website/blog/2023-01-25-ferretdb-fetches-data-query-pushdown.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+Fortunately, we’ve managed to introduce the query pushdown with [this PR](https://github.com/FerretDB/FerretDB/pull/1207)!
+
+## How FerretDB fetches data using query pushdown
+
+Let's jump right into the FerretDB internals and go step by step to see how it handles the sample query!
+
+Let's say we have a collection with data on thousands of customers, and want to check if the one under “john.doe@example.com” email address has an active account:
+
+```js
+db.customers.find({ 'email': 'john.doe@example.com' },{ active: 1 })
```
[the remark this answers is not in the document]
Comment (Fashander) [reply website/blog/2023-01-25-ferretdb-fetches-data-query-pushdown.md:40]: @noisersup PTAL

[the remark this answers is not in the document]
Comment (noisersup) [reply website/blog/2023-01-25-ferretdb-fetches-data-query-pushdown.md:40]: That's the [projection](https://www.mongodb.com/docs/manual/tutorial/project-fields-from-query-results/#return-the-specified-fields-and-the-_id-field-only), not a filter

**Review by w84thesun — APPROVED** [review]

**Review by ptrfarkas — APPROVED** [review]

**Review by w84thesun — APPROVED** [review]
:shipit:

**Review by noisersup — APPROVED** [review]
:shipit:

**Review by ptrfarkas — APPROVED** [review]

**Pull request merged**